### PR TITLE
Gitub Update

### PR DESCRIPTION
**Pull request by keppenj**
Adding in new research, positions, and pictures.

### CODE REVIEW

**Comment by nickwhitman1993** [comment]
@rtrp/admins looks good to me. :shipit:

**Comment by ryanstwrt** [comment]
@keppenj when I took a loot at your page, your `research interests` did not show up as bolded.
It might be due to the fact that the three `#` are not flush with the right hand side, I think there is a space there.

**Comment by ryanstwrt** [comment]
Also, your figure doesn't work. You forgot to add the `.` between the name and `jpg`.

**Comment by nickwhitman1993** [comment]
@ryanstwrt Good catches, second those edits.

**Comment by nickwhitman1993** [comment]
@rtrp/admins Changes were made as outlined above. :shipit:

**Comment by keppenj** [comment]
Fixed. Do I start a new pull request to get this through, now?

File: users/keppenj/index.md [hunk 1]
```diff
@@ -6,10 +6,29 @@ author: Jackson Keppen
 Jackson Keppen
 ================================
 
+<img src="./Images/jackson.jpg" height="375" width="600">
+
 Hello!
 
-I am a junior in Nuclear Engineering at Oregon State University. I recently began work with Dr. Palmer on a Lead-Slowing Down Spectrometer (LSDS) that we hope to acquire from PNNL. 
+I am a second year Master's student at Oregon State University, studying nuclear engineering under Dr. Camille Palmer and Dr. Todd Palmer. I received my bachelors degree in nuclear engineering from Oregon State in 2018. I currently work at NuScale Power as an engineering intern in the Office of Technology. I am also a licensed operator on the Oregon State TRIGA Reactor. I interned at Palo Verde Nuclear Generating Station in the Nuclear Analysis department in 2016, and began working at NuScale as an intern in the Regulatory Affairs department in 2017.
```
Comment (aaronjamesreynolds) [inline users/keppenj/index.md]: Minor: Bachelor's degree instead of bachelors

**Comment by aaronjamesreynolds** [comment]
@keppenj you don't need to start another pull request. Your additional commits are getting placed on this this one automagically

**Comment by aaronjamesreynolds** [comment]
Looks good to me!

**Pull request merged**